operations/model.go

```diff
@@ -155,9 +159,50 @@ func (s *ClientSettings) setupRestCommunicator(ctx context.Context, printMessage
 	if printMessages {
 		printUserMessages(ctx, c, !s.AutoUpgradeCLI)
 	}
+
+	if s.shouldGenerateJWT(ctx, c) {
+		grip.Info("We will attempt to generate a JWT token, to opt out of this, set 'do_not_run_kanopy_oidc' to true in your config file")
```

Comment:
nit: prefer to use `Evergreen CLI` here instead of `we` since it's a little vague (we the evergreen team? we the Evergreen pods? we the Evergreen CLI?)